run.sh

```diff
@@ -2,39 +2,259 @@
+    else
+        kubectl delete namespace $BENCHMARK_NAMESPACE --ignore-not-found=true
+    fi
+}
+
+create_s3_storage() {
+    bucket_names=$1
+
+    if $IS_OPENSHIFT; then
+        echo -e "\nCreating AWS S3 storage"
+        ./hack/scripts/create-s3-bucket.sh $bucket_names
+    fi
+}
+
+destory_s3_storage() {
+    bucket_names=$1
+
+    if $IS_OPENSHIFT; then
+        echo -e "\nDestroying AWS S3 storage"
+        ./hack/scripts/delete-s3-bucket.sh $bucket_names
+    fi
+}
+
+configure_prometheus() {
+    echo -e "\nConfiguring Prometheus"
+
+    if $IS_OPENSHIFT; then
+        enable_ocp_prometheus_monitoring
+        export_ocp_prometheus_settings
+    else
+        echo -e "\nForward ports to loki components"
+        forward_ports
+
+        echo -e "\nSet prometheus relabel regex"
+        set_prometheus_relabel_regex
+
+        echo -e "\nScrape metrics from Loki components"
+        scrape_loki_metrics
+    fi
+}
+
+wait_for_ready_components() {
+    echo -e "\nWaiting for available querier deployment"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "deploy/$LOKI_COMPONENT_PREFIX-querier" --timeout=600s
+
+    echo -e "\nWaiting for available loki query frontend deployment"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "deploy/$LOKI_COMPONENT_PREFIX-query-frontend" --timeout=600s
+
+    echo -e "\nWaiting for available loki distributor deployment"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "deploy/$LOKI_COMPONENT_PREFIX-distributor" --timeout=600s
+
+    echo -e "\nWaiting for available loki ingester statefulset"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "statefulsets/$LOKI_COMPONENT_PREFIX-ingester" --timeout=600s
+
+    echo -e "\nWaiting for available loki index gateway statefulset"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "statefulsets/$LOKI_COMPONENT_PREFIX-index-gateway" --timeout=600s
+
+    echo -e "\nWaiting for available loki compactor statefulset"
+    kubectl -n "$BENCHMARK_NAMESPACE" rollout status "statefulsets/$LOKI_COMPONENT_PREFIX-compactor" --timeout=600s
+}
+
+run_benchmark_suite() {
+    output_directory=$1
+    
+    $GINKGO --output-dir=$output_directory --json-report=report.json --junit-report=report.xml --timeout=4h ./benchmarks
+}
+
+enable_ocp_prometheus_monitoring() {
+    echo -e "\nAdding user workload monitoring configuration"
+
+    kubectl -n openshift-monitoring apply -f $ocp_prometheus_config_path/cluster-monitoring-config.yaml
+	kubectl -n openshift-user-workload-monitoring apply -f $ocp_prometheus_config_path/user-workload-monitoring-config.yaml
+}
+
+disable_ocp_prometheus_monitoring() {
+    echo -e "\nRemoving user workload monitoring configuration"
+
+    kubectl -n openshift-monitoring delete -f $ocp_prometheus_config_path/cluster-monitoring-config.yaml --ignore-not-found=true
+	kubectl -n openshift-user-workload-monitoring delete  -f $ocp_prometheus_config_path/user-workload-monitoring-config.yaml --ignore-not-found=true
+}
```

Comment:
Suggestion to call it with it's real name: `(enable|disable)_ocp_user_workload_monitoring`